converter/src/main/java/org/codice/countrycode/CountryCodeSimple.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.countrycode;
+
+import static org.codice.countrycode.CountryCodeSimple.Format.ALPHA2;
+import static org.codice.countrycode.CountryCodeSimple.Format.ALPHA3;
+import static org.codice.countrycode.CountryCodeSimple.Format.NUMERIC;
+import static org.codice.countrycode.CountryCodeSimple.Standard.FIPS_10_4;
+import static org.codice.countrycode.CountryCodeSimple.Standard.GENC_3_0_0;
+import static org.codice.countrycode.CountryCodeSimple.Standard.ISO_3166_1;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import org.codice.countrycode.converter.Converter;
+import org.codice.countrycode.standard.CountryCode;
+import org.codice.countrycode.standard.StandardProvider;
+import org.codice.countrycode.standard.StandardRegistry;
+import org.codice.countrycode.standard.StandardRegistryImpl;
+
+/** simple interface for the typical use case */
+// we may want to move this to a different package/module/project
+public class CountryCodeSimple {
+
+  private static final String FIPS = "FIPS";
+  private static final String FIPS_VERSION = "10-4";
+
+  private static final String ISO = "ISO3166";
+  private static final String ISO_VERSION = "1";
+
+  private static final String GENC = "GENC";
+  private static final String GENC_VERSION = "3.0.0";
+
+  private static final StandardProvider FIPS_STANDARD;
```

Comment:
Should the constants live on the StandardProvider as opposed to this class?

Reply:
These constants are available through the Standard classes (e.g. Iso3166Standard), but it felt against the spirit of the existing design to reference them directly.